src/scripts/emailUpperCaseMigration.js

```diff
@@ -0,0 +1,145 @@
+import createDbConnection from "../db/connect.js";
+const knex = createDbConnection();
+
+import { subscribeHash } from "../utils/hibp.js";
+import { getSha1 } from "../utils/fxa.js";
+
+/**
+ * MNTOR-2469: One-off script to lowercase email addresses of subscribers and emails before hashing
+ */
+async function subscribeLowerCaseHashToHIBP(emailAddress) {
+  const lowerCasedEmail = emailAddress.toLowerCase();
+  const lowerCasedSha1 = getSha1(lowerCasedEmail);
+  await subscribeHash(lowerCasedSha1);
+  return lowerCasedSha1;
+}
+
+(async () => {
+  const chunkSize = 1000;
+  console.log(
+    `Fixing lower-cased emails / sub to hashes in ${chunkSize}-sized chunks`,
+  );
+
+  console.log(`/** Subscribers Table **/`);
```

Comment:
Hm, why the multi-line comment markers in the log statement?

Reply:
Yeah, this is where the structured logging really helps :) We should get that working with cron jobs soon.